assets/_scss/elements/_typography.scss

```diff
@@ -116,7 +116,7 @@ dfn {
 }
 
 .usa-sans,
-.usa-sans p {
+.usa-sans * {
```

Comment:
Why are we using a wildcard here? Not sure this is best practice and would be hard for users to un-tangle.